src/services/dynamic-styling.service.ts

```diff
@@ -55,9 +54,7 @@ class DynamicStylingService {
    *
    * @param searchResults
    */
-  public getSearchBarAutoCompleteHeight = (
-    searchResults: (POI | Building)[]
-  ) => {
+  public getSearchBarAutoCompleteHeight = (searchResults: any[]) => {
```

Comment:
Why is this of type any when previously it was of type POI | Bulding?

Reply:
Because its type doesn't matter. Only its length is used, no its data. In fact, it would have been better to change that parameter to be just a number, but that wasn't the point of this overhaul.

I changed it to any because I was getting type error from calling this method, and thought it would be better to explicitly ignore whatever data is within that array.

If you want, I can change it to MarkerLocation, altought it won't change anything

Reply:
I would prefer it to be MarkerLocation since searchResults is an array of MarkerLocations. Would this result in the type error you mentioned?

Reply:
Actually, it would be of type SearchResult[], right?

Reply:
Sorry yeah I made a mistake, it would be of type SearchResult.

I don't quite understand why you want it like that since it currently doesn't matter what type the array elements are.

Reply:
This would be nice to have, but not a blocker.